sandbox/settings.py

```diff
@@ -515,6 +515,11 @@ class Base(DRFMixin, Configuration):
         },
     }
 
+    # Elasticsearch
+    RICHIE_ES_HOST = values.Value(
+        "elasticsearch", environ_name="RICHIE_ES_HOST", environ_prefix=None
```

Comment:
Why not just:
```
RICHIE_ES_HOST = values.Value("elasticsearch")
```

Reply:
Because, we want it to be overridable by an environment variable not prefixed by `DJANGO_`. Am I wrong?

Reply:
IC. But since it is a Django setting like the others I think we should follow the same pattern.

Reply:
This is not the case for database-related settings, and in this case, it should be called `ES_HOST`, and thus be overridable by a `DJANGO_ES_HOST` environment variable...

Reply:
but DATABASES is a common Django setting where ES_HOST is just for Richie... Still I'm ok to remove the RICHIE_ if you think it's better, but think of the coherence with all the other richie settings.

Reply:
Don’t get me wrong: I prefer to let it as is, ie named RICHIE_ES_HOST, and let’s make all other Richie specific setting follow the same pattern, ie RICHIE_*

Reply:
:+1:  it should already be the case for all the settings I moved to defaults.py last week.

Reply:
Ok then I'll mark this as resolved.